tests/VenusChainlinkOracleTest.js

```diff
@@ -0,0 +1,102 @@
+const {
+  makeChainlinkOracle,
+  makeVToken,
+} = require('./Utils/Venus');
+
+describe('VenusChainlinkOracle', () => {
+  let root, accounts;
+  let oracle, vBnb, vDai, vUsdc, vUsdt;
+
+  beforeEach(async () => {
+    [root, ...accounts] = saddle.accounts;
+    vBnb = await makeVToken({kind: "vbnb",
+      comptrollerOpts: {kind: "v1-no-proxy"},
+      supportMarket: true
+    });
+    vUsdc = await makeVToken({
+      comptroller: vBnb.comptroller,
+      supportMarket: true,
+      underlyingOpts: {symbol: "USDC"}
+    });
+    vUsdt = await makeVToken({
+      comptroller: vBnb.comptroller,
+      supportMarket: true,
+      underlyingOpts: {symbol: "USDT"}
+    });
+    vDai = await makeVToken({
+      comptroller: vBnb.comptroller,
+      supportMarket: true,
+      underlyingOpts: {symbol: "DAI"}
+    });
+    bnbFeed = await makeChainlinkOracle({decimals: 8, initialAnswer: 30000000000});
+    usdcFeed = await makeChainlinkOracle({decimals: 8, initialAnswer: 100000000});
```

Comment:
can you make different decimal tokens for test like 6, 8, 18?

Reply:
I sure can, but just to clarify the usage here: our Chainlink oracles always store USD-denominated feeds at 8 decimals, regardless of the decimals of the underlying token.

Reply:
I meant, different underlying token decimals(ex: 6, 8, 18). since we use different price decimals for different token decimals.